### PR TITLE
Added Dark Theme to the broadcast section and fixed the social icons

### PR DESCRIPTION
**Pull request by Coder-Srinivas**
## Issue that this pull request solves

 Issue: #490

Added dark theme to the broadcast section and fixed the overlapping social icons.

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

### CODE REVIEW

File: frontend/src/components/util/Toggle/toggle.module.scss [hunk 1]
```diff
@@ -3,6 +3,10 @@
   padding-left: 4rem;
 }
 
+// .mobile-toggle{
```
Comment (Kajol-Kumari) [inline frontend/src/components/util/Toggle/toggle.module.scss:6]: can we please remove it if it's not required

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
The on-hover effect of the social media icons of about-us` page kinda gets hid in dark mode. Can you please change it to a better hover-effect instead

File: frontend/src/components/util/Toggle/toggle.module.scss (hunk 1 above)
Comment (Coder-Srinivas) [reply frontend/src/components/util/Toggle/toggle.module.scss]: Removed it

**Comment by Kajol-Kumari** [comment]
> The on-hover effect of the social media icons of about-us` page kinda gets hid in dark mode. Can you please change it to a better hover-effect instead

Hey! I can't see this change. Can you please update this as well?

**Comment by Coder-Srinivas** [comment]
Hey @Kajol-Kumari, sorry I have overlooked the above message. Will change it ASAP.

**Comment by Kajol-Kumari** [comment]
> Hey @Kajol-Kumari, sorry I have overlooked the above message. Will change it ASAP.

Hey @Coder-Srinivas can you please fix it ASAP

**Comment by Coder-Srinivas** [comment]
@Kajol-Kumari fixed all issues.

**Review by Kajol-Kumari — APPROVED** [review]
LGTM :+1:

**Pull request merged**